src/transactions/TransactionUtils.cpp

```diff
@@ -1951,4 +1951,11 @@ makeClaimAtom(uint32_t ledgerVersion, AccountID const& accountID,
     }
     return atom;
 }
+
+uint64_t
+getTransactionEnvelopeSize(TransactionEnvelope const& envelope)
+{
+    return static_cast<uint64_t>(xdr::xdr_size(envelope));
```

Comment:
Why cast and return this as a `uint64_t`? It looks like you use this function as the original `int64_t` more often.

Reply:
`xdr_size` returns a `size_t` (unsigned), so I cast this to `uint64_t`. IMO generally sizes should be unsigned, and it seems thats how things are in the codebase (e.g. `FlowControlCapacity::msgBodySize` returns `uint64_t`), however the `Resource` class uses `int64_t` to store resource values (but afaik the inputs it stores nearly always unsigned and cast when populating it). I'd rather keep it unsigned here rather than cast unsigned to signed here and back to unsigned elsewhere.